### PR TITLE
fixed the link

### PR DESCRIPTION
**Pull request by Tej-Singh-Rana**
Replaced the link with the correct one.

Regards,

### CODE REVIEW

File: content/en/blog/_posts/2022-04-28-Increasing-the-security-bar-in-Ingress-NGINX/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Linux containers (and underlying mechanisms such as kernel namespaces) work.
 You can read about cgroups in the Kubernetes glossary: [`cgroup`](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cgroup) and learn more about cgroups interact with namespaces in the NGINX project article
 [What Are Namespaces and cgroups, and How Do They Work?](https://www.nginx.com/blog/what-are-namespaces-cgroups-how-do-they-work/).
 (As you read that, bear in mind that Linux kernel namespaces are a different thing from
-[Kubernetes namespaces](h/docs/concepts/overview/working-with-objects/namespaces/)).
+[Kubernetes namespaces](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)).
```
Comment (divya-mohan0209) [inline content/en/blog/_posts/2022-04-28-Increasing-the-security-bar-in-Ingress-NGINX/index.md:48]: @Tej-Singh-Rana, we do not use direct hyperlinking as a standard within the K8s project website. Please could you amend this to the below?
```suggestion
[Kubernetes namespaces](/docs/concepts/overview/working-with-objects/namespaces/)).
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Please see feedback around hyperlinking.

File: content/en/blog/_posts/2022-04-28-Increasing-the-security-bar-in-Ingress-NGINX/index.md (hunk 1 above)
Comment (Tej-Singh-Rana) [reply content/en/blog/_posts/2022-04-28-Increasing-the-security-bar-in-Ingress-NGINX/index.md]: Thanks, didn't know about this. 
Can you please check again? I changed it. 

Regards,

File: content/en/blog/_posts/2022-04-28-Increasing-the-security-bar-in-Ingress-NGINX/index.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Linux containers (and underlying mechanisms such as kernel namespaces) work.
 You can read about cgroups in the Kubernetes glossary: [`cgroup`](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cgroup) and learn more about cgroups interact with namespaces in the NGINX project article
 [What Are Namespaces and cgroups, and How Do They Work?](https://www.nginx.com/blog/what-are-namespaces-cgroups-how-do-they-work/).
 (As you read that, bear in mind that Linux kernel namespaces are a different thing from
-[Kubernetes namespaces](h/docs/concepts/overview/working-with-objects/namespaces/)).
+[Kubernetes namespaces](/docs/concepts/overview/working-with-objects/namespaces/)).
```
Comment (divya-mohan0209) [inline content/en/blog/_posts/2022-04-28-Increasing-the-security-bar-in-Ingress-NGINX/index.md:48]: Extra bracket at the end.
```suggestion
[Kubernetes namespaces](/docs/concepts/overview/working-with-objects/namespaces/).
```

**Review by divya-mohan0209 — COMMENTED** [review]
Extra bracket to be removed at the end, otherwise lgtm!

File: content/en/blog/_posts/2022-04-28-Increasing-the-security-bar-in-Ingress-NGINX/index.md (hunk 2 above)
Comment (Tej-Singh-Rana) [reply content/en/blog/_posts/2022-04-28-Increasing-the-security-bar-in-Ingress-NGINX/index.md]: Thanks

**Comment by divya-mohan0209** [comment]
Hey @Tej-Singh-Rana , I am sorry I just noticed this now. But could you revert to your previous commit if possible? I made a mistake of not checking that the whole link was a part of a bracketed sentence. If not, could you please re-add the bracket?

**Pull request merged**